vertical-pod-autoscaler/pkg/utils/vpablessing/vpa_blessing.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vpablessing
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+const (
+	// VpaBlessingLabel is a label used by the vpa blessing annotation.
+	VpaBlessingLabel = "containersObservedByAdmissionController"
+	opList           = "["
+	endList          = "]"
+	listSeparator    = ", "
+)
+
+// GetVpaBlessingAnnotationValue creates an annotation vlue for a given pod.
+func GetVpaBlessingAnnotationValue(pod *v1.Pod) string {
+	containerNames := make([]string, len(pod.Spec.Containers), len(pod.Spec.Containers))
```

Comment:
nit: just make([]string, len(pod.Spec.Containers)) means the same (capacity is equal to length)

Reply:
Done.